### DISCUSSION TITLE
error running pod install flutter

### DISCUSSION THREAD

**Discussion by sorydima** [codemagic-ci-cd/codemagic-docs#2191] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Clone and try to build https://github.com/sorydima/REChain-.git for iOS Platform on Codemagic.
2. You will see the error in terminal "error running pod install". 
![1](https://github.com/codemagic-ci-cd/codemagic-docs/assets/43252008/d95ac99b-b5dc-472f-baa4-55367f8df0fc)

3. So the solutuion to run on local machine with Apple M-chip is https://stackoverflow.com/questions/54135078/how-to-solve-error-running-pod-install-in-flutter-on-mac
4. How to resolve it in Codemagic remotely?

### Expected results

Successfull build for iOS/Mac OS on Codemagic. 

### Actual results

"Error running pod install"
![1](https://github.com/codemagic-ci-cd/codemagic-docs/assets/43252008/5af4b9b2-8232-4648-9e94-ad6c259a75b8)

### Build id (optional)

654cffb9d0a49a8f66cffb1f

**Comment by icarusdust** [comment, marked as answer]
Hi @sorydima, if you scroll a bit up in the logs, you should be able to see the actual error message. You are getting a similar error as in [this thread](https://github.com/orgs/codemagic-ci-cd/discussions/1682#discussioncomment-5948244) for a different package, and the thread gives you a solution. Let me know if you have any further questions.
